app/models/concerns/news_image.rb

```diff
@@ -0,0 +1,23 @@
+module NewsImage
```

Comment:
I think the concerns should be put in to separate commits and the corresponding tests included too e.g. for this file you can include https://github.com/alphagov/frontend/pull/4573/commits/70b58e793e21872674dac097e78959138b962816 in the same commit.